### ISSUE TITLE
Broken in Wagtail 2.7 – wagtail.utils.pagination is deprecated

### ISSUE THREAD

**Issue by apisitas** [torchbox/wagtailmedia#62]
I got a warning because wagtailmedia/views/chooser.py and wagtailmedia/views/emdia.py still use wagtail pagination. 
Can you update this please? It will be removed in future versions.

**Comment by thibaudcolas** [comment]
Hey @apisitas, thanks for the report. 

We use an [issue template](https://github.com/torchbox/wagtailmedia/issues/new) to make it easier to follow bug reports on this project – could you use it please? Here it is:

```
Found a bug? Please fill out the sections below. 👍

### Issue Summary

A summary of the issue.


### Steps to Reproduce

1. (for example) Start a new project with `wagtail start myproject`
2. Integrate `wagtailmedia` as follows…
3. ...

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### Technical details

* Python version: Run `python --version`.
* Django version: Look in your requirements.txt, or run `pip show django | grep Version`.
* Wagtail version: Hover over the Wagtail bird in the admin, or run `pip show wagtail | grep Version:`.
* `wagtailmedia` version: Look in your requirements.txt, or run `pip show wagtailmedia | grep Version`.
* Browser version: You can use http://www.whatsmybrowser.org/ to find this out.
```

**Comment by Chris-May** [comment]
### Issue Summary

A warning is raised whenever one runs a `manage.py` command. The warning is a `RemovedInWagtail27Warning`: 
```wagtail.utils.pagination is deprecated. Use django.core.paginator.Paginator directly with get_page instead category=RemovedInWagtail27Warning)```

### Steps to Reproduce

1. Start a new project with `wagtail start myproject`
2. Integrate `wagtailmedia` as the directions 
3. Run any `manage.py` script. The warning will appear quickly.

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

I considered this a bug because it caused build scripts to fail, only because of the warning message.

### Technical details

* Python version: Python 3.7.2
* Django version: 2.2.6
* Wagtail version: 2.6.2
* `wagtailmedia` version: 0.3.1
* Browser version: N/A

**Comment by andsimakov** [comment]
Now it is unusable with recently released Wagtail 2.7 due to promised `wagtail.utils.pagination` deprecation.

**Comment by thibaudcolas** [comment]
@Chris-May in #63 has started work on updating wagtailmedia to not rely on this util. I think it would be helpful if more people tested his fix and report back whether there are other issues. 

I don’t have time to further look into it myself but will try to commit some time in a week or so, hopefully take care of the Wagtail 2.6 / 2.7 compatibility, make a release, and take a stab at #48.

**Comment by thenewguy** [comment]
@apisitas tests run and pass against Wagtail 2.7 now.  Please close this issue if it is resolved for you

**Comment by andsimakov** [comment]
> @apisitas tests run and pass against Wagtail 2.7 now. Please close this issue if it is resolved for you

When the version will be updated to be installable via pip? I never faced a procedure like this.

**Comment by thenewguy** [comment]
@andsimakov my mistake. I did not realize a new version had not been released yet.  I cannot speak as to when they will do so

**Comment by zerolab** [comment]
@andsimakov until a new release is made, you can add `git+git@github.com:torchbox/wagtailmedia.git@54c1eb8fe98534b1b6db8391b24a7b1218a7114e#egg=wagtailmedia` to your requirements txt

hope it helps

**Comment by dripwalking** [comment]
@zerolab I try to add your path to my requirements.txt but get an error:
`pip3 install -r requirements.txt`

`ERROR: Invalid requirement: 'git+git@github.com:torchbox/wagtailmedia.git@54c1eb8fe98534b1b6db8391b24a7b1218a7114e#egg=wagtailmedia' (from line 3 of requirements.txt)`
`Hint: It looks like a path. File 'git+git@github.com:torchbox/wagtailmedia.git@54c1eb8fe98534b1b6db8391b24a7b1218a7114e#egg=wagtailmedia' does not exist.`

**Comment by webconexus** [comment]
> @zerolab I try to add your path to my requirements.txt but get an error:
> `pip3 install -r requirements.txt`
> 
> `ERROR: Invalid requirement: 'git+git@github.com:torchbox/wagtailmedia.git@54c1eb8fe98534b1b6db8391b24a7b1218a7114e#egg=wagtailmedia' (from line 3 of requirements.txt)`
> `Hint: It looks like a path. File 'git+git@github.com:torchbox/wagtailmedia.git@54c1eb8fe98534b1b6db8391b24a7b1218a7114e#egg=wagtailmedia' does not exist.`

@dripwalking 

Add `-e` before the path. Like so:

```
-e git+git@github.com:torchbox/wagtailmedia.git@54c1eb8fe98534b1b6db8391b24a7b1218a7114e#egg=wagtailmedia
```

And remove the original `wagtailmedia` requirement.

**Comment by dripwalking** [comment]
@webconexus Thank you a lot for help!

**Comment by webconexus** [comment]
@zerolab I seem to get an internal server error when trying to use the `MediaChooserPanel` in the Wagtail admin. 

```
http://localhost:8000/cms/media/chooser/ 500 (Internal Server Error)
send @ jquery-3.2.1.min.js?v=ca17d535:4
ajax @ jquery-3.2.1.min.js?v=ca17d535:4
r.<computed> @ jquery-3.2.1.min.js?v=ca17d535:4
ModalWorkflow.self.loadUrl @ modal-workflow.js?v=ca17d535:33
ModalWorkflow @ modal-workflow.js?v=ca17d535:87
(anonymous) @ media-chooser.js:8
dispatch @ jquery-3.2.1.min.js?v=ca17d535:3
q.handle @ jquery-3.2.1.min.js?v=ca17d535:3
```

There seems to be a problem with jQuery?

**Comment by zerolab** [comment]
fixed in 0568167

**Comment by zerolab** [comment]
Have finally release [v0.4.0](https://pypi.org/project/wagtailmedia/0.4.0/)
